src/Domain/Package/PackageManager.php

```diff
@@ -181,7 +181,7 @@ private function initializePackages(FixturePathHandler $fixture_path_handler, st
       // Skipping a null datum provides for a package to be effectively removed
       // from the active specification at runtime by setting its value to NULL
       // in the packages configuration alter file.
-      if ($datum === NULL) {
+      if ($datum === NULL || (array_key_exists('core_matrix', $datum) && $datum['core_matrix'] === NULL)) {
```

Comment:
Wouldn't this (incorrectly) cause the following valid package to be removed? Better add a test case.

```yaml
drupal/example:
  core_matrix: ~
  version: 1.x
  version_dev: 1.x-dev
```

Reply:
We cannot keep core_matrix as NULL as it will throw an error with the OptionsResolver where core_matrix is typed as an array.

Reply:
Oh, I see. Great; that's fine, then.